content/client-server-api/modules/instant_messaging.md

```diff
@@ -37,9 +37,24 @@ HTML injection, and similar attacks. The strongly suggested set of HTML
 tags to permit, denying the use and rendering of anything else, is:
 `font`, `del`, `h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `blockquote`, `p`,
 `a`, `ul`, `ol`, `sup`, `sub`, `li`, `b`, `i`, `u`, `strong`, `em`,
-`strike`, `code`, `hr`, `br`, `div`, `table`, `thead`, `tbody`, `tr`,
+`strike`, `s`, `code`, `hr`, `br`, `div`, `table`, `thead`, `tbody`, `tr`,
 `th`, `td`, `caption`, `pre`, `span`, `img`, `details`, `summary`.
 
+
+{{% boxes/note %}}
+HTML features MAY be deprecated and replaced by their modern equivalent without
+requiring a [Spec Change Proposal](/proposals) when they are deprecated in the
+WHATWG HTML Living Standard.
+{{% /boxes/note %}}
+
+{{% boxes/note %}}
+{{% changed-in v="1.10" %}}
+
+The `strike` tag is deprecated. Clients MUST stop sending new messages using
+this tag and replace it with `s` or `del`.
+{{% /boxes/note %}}
```

Comment:
```suggestion
```

The MSC does not permit this strong of language. What we can do is simply remove `strike`, like in https://github.com/matrix-org/matrix-spec/pull/1629

Reply:
Removed